samples/Mvc.Client/Startup.cs

```diff
@@ -49,6 +49,13 @@ public class Startup {
                 options.ClientSecret = "98f1bf028608901e9df91d64ee61536fe562064b";
             });
 
+            app.UseFourSquareAuthentication(options => { 
```

Comment:
Is this the official casing of `Foursquare`? Looks like it's not an uppercase S: https://en.wikipedia.org/wiki/Foursquare